src/views/Pools/components/PoolCard.tsx

```diff
@@ -44,167 +31,47 @@ const PoolCard: React.FC<HarvestProps> = ({ pool }) => {
-        <StyledCardActions>
-          {!account && <UnlockButton />}
-          {account &&
-            (needsApproval && !isOldSyrup ? (
-              <div style={{ flex: 1 }}>
-                <Button disabled={isFinished || requestedApproval} onClick={handleApprove} width="100%">
-                  {`Approve ${stakingToken.symbol}`}
-                </Button>
-              </div>
-            ) : (
-              <>
-                <Button
-                  disabled={stakedBalance.eq(new BigNumber(0)) || pendingTx}
-                  onClick={
-                    isOldSyrup
-                      ? async () => {
-                          setPendingTx(true)
-                          await onUnstake('0', stakingToken.decimals)
-                          setPendingTx(false)
-                        }
-                      : onPresentWithdraw
-                  }
-                >
-                  {`Unstake ${stakingToken.symbol}`}
-                </Button>
-                <StyledActionSpacer />
-                {!isOldSyrup && (
-                  <IconButton disabled={isFinished && sousId !== 0} onClick={onPresentDeposit}>
-                    <AddIcon color="white" />
-                  </IconButton>
-                )}
-              </>
-            ))}
-        </StyledCardActions>
-        <StyledDetails>
-          <div>{TranslateString(736, 'APR')}:</div>
-          {isFinished || isOldSyrup || !apy ? (
-            '-'
-          ) : (
-            <Balance fontSize="14px" isDisabled={isFinished} value={apy} decimals={2} unit="%" />
-          )}
-        </StyledDetails>
-        <StyledDetails>
-          <div>{TranslateString(384, 'Your Stake')}:</div>
-          <Balance
-            fontSize="14px"
-            isDisabled={isFinished}
-            value={getBalanceNumber(stakedBalance, stakingToken.decimals)}
-          />
-        </StyledDetails>
-      </div>
+      <CardHeader
+        title={`${TranslateString(318, 'Earn')} ${tokenName}`}
+        coinIconUrl={`/images/pools/${poolImage}`}
+        tokenName={tokenName}
+        stakingTokenName={stakingToken.symbol}
+        isFinished={isFinished && sousId !== 0}
+      />
+      <Flex flexDirection="column" padding="24px">
+        <Apr tokenName={tokenName} isOldSyrup={isOldSyrup} isFinished={isFinished} apy={apr} />
+        <Earned
+          isFinished={isFinished}
+          sousId={sousId}
+          earningTokenName={earningToken.symbol}
+          stakingTokenName={stakingToken.symbol}
+          isBnbPool={isBnbPool}
+          harvest={harvest}
+          isOldSyrup={isOldSyrup}
+          earnings={earnings}
+          earningTokenDecimals={tokenDecimals}
+        />
+        <Stake pool={pool} isOldSyrup={isOldSyrup} isBnbPool={isBnbPool} />
+      </Flex>
       <CardFooter
         projectLink={earningToken.projectLink}
         decimals={stakingToken.decimals}
```

Comment:
again, here also. it's indescriptive. `stakingTokenDecimals={stakingToken.decimals}`